### PR TITLE
update commit msg to make not user facing valid in itself

### PR DESCRIPTION
**Pull request by albanD**

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: .github/scripts/process_commit.py [hunk 1]
```diff
@@ -28,8 +28,10 @@
     "topic: performance",
     "topic: documentation",
     "topic: developer feature",
-    "topic: non-user visible",
+    "topic: not user facing",
```
Comment (anjali411) [inline .github/scripts/process_commit.py:31]: lol

File: .github/scripts/process_commit.py [hunk 2]
```diff
@@ -71,13 +73,14 @@ def get_repo_labels() -> List[str]:
...
-    for the 'release notes: ...' and [here](https://github.com/pytorch/pytorch/labels?q=topics) for the
-    'topics: ...'."""}
+You've committed this PR, but it does not have both a 'release notes: ...' and 'topics: ...' label. \
+Please add one of each to the PR. The 'release notes: ...' label should represent the part of \
+PyTorch that this PR changes (fx, autograd, distributed, etc) and the 'topics: ...' label should \
+represent the kind of PR it is (not user facing, new feature, bug fix, perf improvement, etc). \
+The list of valid labels can be found [here](https://github.com/pytorch/pytorch/labels?q=release+notes) \
+for the 'release notes: ...' and [here](https://github.com/pytorch/pytorch/labels?q=topic) for the \
+'topics: ...'.
+For changes that are 'topic: not user facing' there is no need for a release notes label."""}
```
Comment (anjali411) [inline .github/scripts/process_commit.py:83]: release notes category label

**Review by anjali411 — APPROVED** [review]

**Comment by albanD** [comment]
Trying again from a branch in pytorch/pytorch

**Pull request closed without merging**